app/pages/index.tsx

```diff
@@ -98,12 +145,32 @@ const Home: NextPage = () => {
         </div>
         {!connectedWallet ? (
           <WalletMultiButton className="btn btn-primary" />
-        ) : creator ? (
-          <p>CREATOR FOUND</p>
-        ) : (
-          <button className="btn btn-primary" onClick={handleBecomeCreator}>
-            Become a Creator
+        ) : user.creatorAccount ? (
+          <button
+            className="btn btn-primary"
+            onClick={() => router.push('/creator-hub')}
+          >
+            Visit Creator Hub
           </button>
+        ) : (
+          <div>
+            <button className="btn btn-primary" onClick={handleBecomeCreator}>
+              Become a Creator
+            </button>
+            <br />
+            <button
+              className="btn btn-primary"
+              onClick={() => {
+                user.subscriptions.length
+                  ? router.push('/subscription-hub')
+                  : router.push('/subcribe');
+              }}
+            >
+              {user.subscriptions.length
+                ? 'See my subsriptions'
+                : 'Subscribe to a creator'}
```

Comment:
For this we can ignore the case if the user has no subscriptions. So if they don't have subscriptions they just see the one button for "Become a Creator"

Reply:
The user should be able to navigate right ? to their subscriptions ?

Reply:
Or should we put those things in a menu item ?

Reply:
Correct, we'll have a page where a user can see all of their subscriptions.

Reply:
yes I believe I am creating that page -> subsription-hub. I meant to say how would they access that page ? Like a navigation

Reply:
The landing page should have a button for that, which is this one for "See my subscriptions". And the CreatorHub page will have a link to it as well, in case a creator wants to get there